### PR TITLE
Expanded the supported php versions php ^7.2 and ^8.0

### PR DESCRIPTION
**Pull request by fuyutsuki**
Title says all

### CODE REVIEW

**Review by supercrafter333 — CHANGES_REQUESTED** [review]
PocketMine-MP does not support PHP 8! PocketMine-MPs min PHP version is PHP 7.3.

File: virion.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: apibossbar
 version: 0.0.5
 antigen: xenialdan\apibossbar
-api: [3.13]
-php: [7.2]
+api: 3.13
+php:
+  - 7.2
+  - 8.0
```
Comment (supercrafter333) [inline virion.yml:7]: Remove 7.2 and 8.0 and set it to 7.3

**Review by supercrafter333 — CHANGES_REQUESTED** [review]

File: virion.yml (hunk 1 above)
Comment (fuyutsuki) [reply virion.yml:7]: Are you serious? This PR allows FormAPI to be used under pmmp's experimental php8 support.
Also, it is designed for php7.2.
so I, as a non-author, should not update it from 7.2 to 7.3 without checking the code…

Comment (supercrafter333) [reply virion.yml:7]: PHP 8 has more changes as PHP 7.3. You're lost. You open a fucking PR for adding PHP 8 to php needed versions without checking the code. WTF

Comment (fuyutsuki) [reply virion.yml:7]: ????????

The stable branch of pmmp supports `^7.3 || ^8.0`, what do you think about that?

Comment (fuyutsuki) [reply virion.yml:7]: > PS: You are not a maintainer or contributor to this repository. You are irrelevant.
> The maintainer's opinion, not yours, takes precedence and does not need to be reviewed.

**Comment by fuyutsuki** [comment]
Outdated

**Pull request closed without merging**